hw_diag/utilities/blockchain.py

```diff
@@ -15,7 +15,14 @@ def get_helium_blockchain_height():
     Possible exceptions:
     TypeError - if the key ['data']['height'] in response is not found.
     """
+    headers = {
+                "user-agent": ("Mozilla/5.0 (X11; U;"
+                               " Linux i686; it; rv:1.9.0.10)"
+                               " Gecko/2009042513 Ubuntu/8.04"
+                               " (hardy) Firefox/3.0.10"),
+              }
```

Comment:
@SebastianMaj let's use a more modern string, like: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15`

Reply:
@vpetersson Done it's now updated, I'll push it to the miner and report results